lib/MusicBrainz/Server/Entity/LinkAttributeType.pm

```diff
@@ -78,7 +78,7 @@ around TO_JSON => sub {
     return {
         %{ $self->$orig },
         gid => $self->gid,
-        root_id => $self->root_id + 0,
+        root_id => defined $self->root_id ? $self->root_id + 0 : 0,
```

Comment:
In 6b0254a65c53a3b2ca3fbfce0fdc1d92ad52edc2 you used defined () with parens. I'm ok with either, but we can try and be consistent :)

Reply:
Parenthesis are not needed. Amended 6b0254a65c with d102a748e1 (see below).